lib/parliament/data/api/v1/party_index/index

```diff
@@ -1 +1 @@
-{ "statusCode": 429, "message": "Rate limit is exceeded. Try again in 1 seconds." }
```

Comment:
Can we make a request that actually returns data here? Or is this here for testing purposes?